engines/bastion/app/views/bastion/layouts/assets.html.erb

```diff
@@ -19,6 +19,7 @@
     angular.module('Bastion').value('currentLocale', '<%= I18n.locale %>');
     angular.module('Bastion').value('CurrentOrganization', "<%= Organization.current.id if Organization.current %>");
     angular.module('Bastion').value('simpleContentAccessEnabled', <%= Organization.current.simple_content_access? if Organization.current %>);
+    angular.module('Bastion').value('isManifestImported', <%= !!Organization.current&.manifest_imported?(cached: true) %>)
```

Comment:
The `!!` is now vestigial thanks to your change

Reply:
I almost deleted it. But if `Organization.current` returns `nil`, without the `!!` this may evaluate to `nil` due to the `&.`, and the  `!!` guarantees it'll be a boolean.

Reply:
in other words I didn't feel like adding `if Organization.current` at the end.

Reply:
The teacher becomes the pupil! That's fair enough